### ISSUE TITLE
Pictures of the HW mods

### ISSUE THREAD

**Issue by raduprv** [NRollo/ESP32CamTimeLapse#1]
Hello, nice project!
Would you please be so kind to post pictures of the actual board and how to do the modifications? The schematics are nice, but pictures would be better.
Thanks!

**Comment by NRollo** [comment]
@raduprv there are some nice pictures here [https://github.com/espressif/esp32-camera/issues/247#issuecomment-1996880238](url). On that page there is also a link to a very nice PCB layout of the ESP32-CAM module, so you should be able to find the tracks and components.
My mods:
![IMG_2994](https://github.com/NRollo/ESP32CamTimeLapse/assets/55288823/f1c45c8e-cc55-4c98-9f68-cb69ad52da20)
I don't have a picture of the changed voltage regulator, but its more a question about de-soldering the original one and solder in the new one.

Enjoy!

**Comment by raduprv** [comment]
Thank you very much. I've actually seen that thread, but was afraid to try it out because some guy there said it didn't work for his board.
So, except for the LDO, and your battery bank hack, this was the entire modification?
I didn't get the chance to look at your code thoroughly, but is there any extra thing you have to do in the code to use the camera compared to the normal way?

Also, I don't have a MCP1702-33, and looking at the specs it provides only 250 mA, which might not be enough for camera and WiFi at the same time, especially if I want to add anything else, like a LORA module or something. I have a DC DC board, I guess I can replace the LDO with that, and then not burn the extra voltage. https://vi.aliexpress.com/item/32799328725.html?spm=a2g0o.order_list.order_list_main.83.28981802JA5w54&gatewayAdapt=glo2vnm

Do you see any problem with this?

**Comment by NRollo** [comment]
The goal for me was to keep the camera running for as long as possible on battery power, as the camera was making a timelapse video of a construction site which I didn't have access to for month. I don't use wifi and I didn't have access to mains, so a battery was the only option. If you need more power than 250mA then yes you should choose another LDO. The updated schematic in the repo should show my mods as well as my suggested mods (doted lines), if even less power consumption is needed.
I haven't "played" with step-up converters and can't say what to look out for, but I guess it should work.
As someone pointed out, in order to get consistent pictures a reset of the camera between pictures is needed (you will find this in the code as well) - don't know why. I am not impressed with the lighting sensitivity of the camera, the pictures get very noisy in the afternoon.

Other than that, have fun - go for it!  :o)

**Comment by raduprv** [comment]
Yes, I know that at night it can be very noisy. This is because the AE algorithm won't prioritize long exposure, so it will instead just up the gain, which is noisy. There are some registry tricks you can use to mitigate that issue, by basically doing a manual exposure. However, you need a real light sensor, because getting the light from the registry is not good enough.
I played with a ov2640 sensor and managed to get it to do good pictures at night (but the tradeoff is that it doesn't work so well during the day). https://github.com/raduprv/esp32-cam_ov2640-timelapse There are similar things you can do with the 5MP sensor, but the registries are very different. I used to work at it a few years ago, but didn't finish it. Maybe I can resume work soon.
